theia-ide/theia-apps
# go: update go images to use vscode extensions

#### What it does


- updates the `theia-go:latest` and `theia-go:next` images to use builtin and external extensions.
- refactor the `download-adapters` script to instead use the `download:plugins` script to both pull the builtins and the [vscode-golang extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.Go)

#### How to test
Build and verify the changes for both `theia-go:latest` and `theia-go:next` images
(that plugins are downloaded and picked up when started)

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)


Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>
